doc/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,16 @@
+[[index]]
+= SmallRye JWT Documentation
+
+SmallRye JWT is an implementation of https://github.com/eclipse/microprofile-jwt-auth/[Eclipse MicroProfile JWT RBAC].
+Currently it is focused on supporting the MP-JWT 1.1 spec, and primarily deals with the parsing of the JWT string into
+a JsonWebToken implementation.
+
+In the future, when MP-JWT 2.0 can build on the JSR-375 security APIs, there should be more support for defining the CDI extension and security layer integration.
+
+== Runtimes
+
+Runtimes that currently use it as their JWT RBAC implementation include:
+
+* https://quarkus.io/[Quarkus]
```

Comment:
@radcortez Is this section recommended across all of Smallrye ? @darranl Are you OK with WildFly referenced ?

Reply:
Other Smallrye projects documentation do this reference. I see no harm in having it, but we can remove it.

Reply:
@radcortez Sure it is a good idea in general; just would like a confirmation he is fine with WildFly being referenced